R/data-set.R

```diff
@@ -363,31 +363,9 @@ DataSet <- R6::R6Class(
       invisible(self)
     },
     .createDataRepository = function() {
-      # Create an empty data repository with a base grid and columns
-      dataRepository <- DataRepository$new()
-
-      # Passing time for dimension for now
-      xColumn <- DataColumn$new(
```

Comment:
@PavelBal Thsi is what I mean when I say we need to move some stuff to .net. in R, we should aim at the minimum amount of marshalling